jekyll/_cci2/runner-server.md

```diff
@@ -46,7 +46,7 @@ A specific server version works with a specific runner version. The table below
 
 Server Version  | Runner
 ----------------|---------------------------------
-3.0 | TBD
```

Comment:
🤔 we'll want this to say 3.1, but I'm not sure if customers are supposed to know about 3.1 yet